process/notifyProcessor.go

```diff
@@ -6,32 +6,35 @@ import (
 	"github.com/ElrondNetwork/node-monitoring/data"
 )
 
-type ArgsNotifyProcessor struct {
-}
-
 type notifyProcessor struct {
 	workers    map[string]Notifier
 	mutWorkers sync.RWMutex
 }
 
-func NewNotifyProcessor(args ArgsNotifyProcessor) (*notifyProcessor, error) {
-	bp := &notifyProcessor{
+// NewNotifyProcessor will create a new notify processor instance
+func NewNotifyProcessor() *notifyProcessor {
+	return &notifyProcessor{
 		workers: make(map[string]Notifier),
 	}
-
-	return bp, nil
 }
 
-func (bp *notifyProcessor) AddNotifier(notifier Notifier) {
-	bp.mutWorkers.RLock()
-	bp.workers[notifier.GetID()] = notifier
-	bp.mutWorkers.RUnlock()
+// AddNotifier will add a notifier instance to workers list
+func (np *notifyProcessor) AddNotifier(notifier Notifier) {
+	np.mutWorkers.Lock()
+	np.workers[notifier.GetID()] = notifier
+	np.mutWorkers.Unlock()
 }
 
-func (bp *notifyProcessor) PushMessage(msg data.NotificationMessage) {
-	bp.mutWorkers.RLock()
-	for _, worker := range bp.workers {
+// PushMessage will push notification message to all registered workers
+func (np *notifyProcessor) PushMessage(msg data.NotificationMessage) {
+	np.mutWorkers.Lock()
```

Comment:
RLock because you are just reading from the map.

Reply:
:+1: